package.json

```diff
@@ -5,16 +5,18 @@
   "homepage": "http://passmarked.com",
   "preferGlobal": true,
   "engines": {
-    "node": ">=0.12.0"
+    "node": ">=0.12"
   },
   "bin": {
     "passmarked": "index"
   },
   "scripts": {
-    "test": "mocha"
+    "test": "mocha",
+    "coverage": "istanbul cover _mocha --report lcovonly -- test -R spec"
   },
   "author": "Passmarked <info@passmarked.com>",
   "contributors": [
```

Comment:
Yeah linking would be nice for me, so we don't have to keep updating package.json too :)